xlators/mgmt/glusterd/src/glusterd-store.c

```diff
@@ -822,11 +822,12 @@ glusterd_volume_exclude_options_write(int fd, glusterd_volinfo_t *volinfo)
     GF_VALIDATE_OR_GOTO(this->name, (conf != NULL), out);
 
     ret = snprintf(buf + total_len, sizeof(buf) - total_len,
-                   "%s=%d\n%s=%d\n%s=%d\n%s=%d\n%s=%d\n",
+                   "%s=%d\n%s=%d\n%s=%d\n%s=%d\n%s=%d\n%s=%d\n",
```

Comment:
I think we need to keep stripe_count in volfile only while op_version is less than (GD_OP_VERSION_10_0),In case of fresh installation we don't need to keep stripe_count.

Reply:
Done